testnet/bitcoin-neon-controller/src/main.rs

```diff
@@ -47,19 +47,31 @@ async fn main() -> http_types::Result<()> {
             Err(err) => err.duration(),
         }.as_secs() as u64;
 
-        let time_since_genesis = now - config.neon.genesis_timestamp;
-        
+        let genesis_timestamp = match std::env::var("NEON_GENESIS_TIMESTAMP") {
+            Ok(val) =>  match val.parse::<u64>() {
+                Ok(val) => val,
+                _ => config.neon.genesis_timestamp,
```

Comment:
Should emit a warning here if the parse fails.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/commit/f2dedf7dbc18e138d0cc1aadfc1c64c9b3cc2f34